interfaces/repo.go

```diff
@@ -593,6 +594,7 @@ func (r *Repository) Connect(ref *ConnRef, plugStaticAttrs, plugDynamicAttrs, sl
 	conn := &Connection{Plug: cplug, Slot: cslot}
 	r.slotPlugs[slot][plug] = conn
 	r.plugSlots[plug][slot] = conn
+	logger.Trace("Interface connection", "interface", slot.Interface, "slot", slot.Snap.SnapType, "plug", plug.Snap.SnapType)
```

Comment:
again, not sure, should this use:

Interface connection
Interface Connection
InterfaceConnection
interface-connection

?